portfolio/src/main/webapp/index.html

```diff
@@ -9,11 +9,32 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p>Hi everyone! My name is Jennifer Guerrero, and I am a sophomore at the University of Illinois at 
+          Urbana-Champaign. I am of Mexican/Honduran heritage, and I was born and raised in Chicago. I am a CSSI
+          alum, and I am very excited to learn more about CS through SPS and to meet new people.
+      </p>
+      <div id= "hobbies">
+        <h2>Hobbies/Interests</h2>
+        <p>I love coffee and going out to try new cafes wherever I go. I am also a semi-successful plant mom with lots
+            of pictures to share that I've taken of my plant endevours. </p>
+        <p>Click <a href="/personal_pictures">here</a> to see some of pictures of the drinks I've tried. </p>
+        <p>Click <a href="/plants.html">here</a> to see some quality green content. </p>
+      </div>
+      <p>Click here to get a random fact about me:</p>
+      <button onclick="addRandomFact()">fact</button>
+      <div id="greeting-container">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <img src="/images/uiuc_logo.jpg" width="100%%" height="100%">
+      </div>
+      <div class="column">
+        <img src="/images/birthday.jpg" width="100%%" height="100%">
+      </div>
+      <div class="column">
+        <img src="/images/chicago.jpg" width="100%%" height="100%">
```

Comment:
Did you want these in personal_pictures.html?

Reply:
Did it in the Jennifer_branch_week_2